### PR TITLE
Clean up Admin Config and Nav

### PR DESCRIPTION
**Pull request by sslaws**
# Fixes or Implements [AB#12894](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/12894)

## Description

Cleanup Configuration and Nav

- Claims page renamed to UserInfo to reflect what it does
- Enabled some Nav items by server configuration
- Enabled UserInfo by server configuration

Add Admin to the Prune Image screen

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

-   [ ] Unit Tests Updated
-   [ ] Functional Tests Updated
-   [X] Not Required

### UI Changes

Yes but minor

### Browsers Tested

-   [X] Chrome
-   [ ] Safari
-   [ ] Edge
-   [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant. Include any specialized deployment steps here.

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

-   Fulfills Work Item Requirements

    -   The changes meet/implement story's acceptance criteria. Fixes identified problems if bug.
    -   Changes not directly related to the task requirements need to be documented on the PR.

-   Compilation / Tests

    -   The changes do not break compilation and/or unit or functional tests; unless other item addresses them specifically.

-   Logic Problems / Functional Behaviour

    -   The changes work as intended and do not introduce problems.

-   Performance

    -   The changes do not introduce obvious performance issues.

-   Documented Standards

    -   The changes adhere to the team's documented [Coding Standards](https://github.com/bcgov/healthgateway/wiki/standards).

-   Readability / Maintainability
    -   The changes can be easily understood/read and allow for future enhancements without major refactoring. Readability is preferred over "clever code".
    -   Reasoning for changes needs to be clearly articulated. Disagreements should be arbitrated by a third developer.

### CODE REVIEW

File: Apps/Admin/Common/Models/ExternalConfiguration.cs [hunk 1]
```diff
@@ -25,25 +24,21 @@ namespace HealthGateway.Admin.Common.Models
...
-        public WebClientConfiguration WebClient { get; set; } = new WebClientConfiguration();
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "Team decision")]
+        public Dictionary<string, bool> Features { get; set; } = new Dictionary<string, bool>();
 
         /// <summary>
         /// Gets or sets the Service Endpoints.
         /// </summary>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "Team decision")]
-        public Dictionary<string, System.Uri> ServiceEndpoints { get; set; } = new Dictionary<string, Uri>();
+        public Dictionary<string, System.Uri> ServiceEndpoints { get; set; } = new Dictionary<string, System.Uri>();
```
Comment (BrianMaki) [inline Apps/Admin/Common/Models/ExternalConfiguration.cs:42]: Why prefix System instead of importing with 'using'?

**Review by BrianMaki — APPROVED** [review]

File: Apps/Admin/Client/Pages/UserInfo.razor [hunk 2]
```diff
@@ -0,0 +1,30 @@
+@page "/userinfo"
```
Comment (MikeLyttle) [inline Apps/Admin/Client/Pages/UserInfo.razor:1]: We should use kebab case for URLs: `/user-info`.

File: Apps/Admin/Client/Layouts/NavMenu.razor [hunk 3]
```diff
@@ -34,7 +35,8 @@
     <AuthorizeView Roles="@Roles.Admin">
         <MudNavLink Href="support" Icon="@Icons.Material.Filled.PersonSearch">Support</MudNavLink>
     </AuthorizeView>
+    @if(ConfigurationState.Value.Result?.Features["VaccineCard"] ?? false){
```
Comment (MikeLyttle) [inline Apps/Admin/Client/Layouts/NavMenu.razor:38]: Braces and spacing are not following our conventions in this file.

**Review by MikeLyttle — CHANGES_REQUESTED** [review]

**Review by MikeLyttle — DISMISSED** [review]

File: Apps/Admin/Client/Layouts/NavMenu.razor (hunk 3 above)
Comment (sslaws) [reply Apps/Admin/Client/Layouts/NavMenu.razor:38]: fixing

File: Apps/Admin/Client/Pages/UserInfo.razor (hunk 2 above)
Comment (sslaws) [reply Apps/Admin/Client/Pages/UserInfo.razor:1]: fixing

**Review by BrianMaki — DISMISSED** [review]

**Review by MikeLyttle — APPROVED** [review]

**Review by MikeLyttle — APPROVED** [review]

File: Apps/Admin/Common/Models/ExternalConfiguration.cs (hunk 1 above)
Comment (sslaws) [reply Apps/Admin/Common/Models/ExternalConfiguration.cs]: Fixed

**Review by BrianMaki — APPROVED** [review]

**Pull request merged**